mcsema/BC/Function.cpp

```diff
@@ -452,6 +452,29 @@ static void CreateLandingPad(TranslationContext &ctx,
     args[0] = ir.CreateLoad(ctx.cfg_func->stack_ptr_var, true);
     args[1] = ir.CreateLoad(ctx.cfg_func->frame_ptr_var, true);
     args[2] = ir.CreateLoad(var_value, true);
+#endif
+    auto handler = GetExceptionHandlerPrologue();
+    ir.CreateCall(handler, args);
+  } else {
+    auto get_index_func = gModule->getFunction("__mcsema_get_type_index");
```

Comment:
Wrap this into a function that gets you the `llvm::Function *` for `__mcsema_get_type_index`